docs/notebooks/demo.ipynb

```diff
@@ -98,18 +126,18 @@
     "        elif self == EmailActions.NEW_PROPOSAL:\n",
     "            email.new_proposed_action()\n",
     "\n",
-    "def reward_func(x: NDArray[np.float64]) -> NDArray[np.float64]:\n",
-    "    return x[..., 0]\n"
+    "def reward_func(samples: NDArray[np.float64]) -> NDArray[np.float64]:\n",
+    "    return samples[..., conversion_column_index]\n"
    ]
   },
   {
    "attachments": {},
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Now we can define our learning agent. To do so, we will need the main decorator from the `bayesianbandits` library - `@bandit`. The `@bandit` decorator synthesizes the bandit API on our experiment class.\n",
+    "Now we can define our learning agent. We first define the arms. Each arm requires an `action_token` (the token that is returned when that particular arm is pulled), a reward function and a learner. This `learner` defines how each arm will learn how rewarding its action is, \n",
     "\n",
-    "The `@bandit` takes two arguments, the `learner` which defines how each arm will learn how rewarding its action is, and `policy`, which determines how the agent chooses which arm to pull. In this case, we'll use `thompson_sampling`, which takes no parameters."
+    "With those arms, we can then define an `Agent`. An `Agent` is defined by a list of arms and a policy, which determines how the agent chooses which arm to pull. In this case, we'll use `thompson_sampling`, which takes no parameters."
```

Comment:
We're using the `ThompsonSampling` class now, correct? Not the old `thompson_sampling` closure function?

Reply:
Corrected